packages/test/test-service-load/testConfig.json

```diff
@@ -12,16 +12,14 @@
             "faultInjectionMaxMs": 150000,
             "optionOverrides":{
                 "odsp":{
-                    "loader":{
-                        "summarizeProtocolTree": [false]
+                    "configurations":{
+                        "Fluid.Container.summarizeProtocolTree": [false]
```

Comment:
I believe Gary's changes hit prod, so there should be no difference between SPDF and PROD.

BTW, two unrelated questions:
- do we have a way to specify options for both SPDF & prod without duplicating them?
- should we rename odsp to reflect that it mentions PROD ring? Maybe it's fine (as we use same naming in other places).

Reply:
Enabled for prod as well. For #2, i think it is fine as it is> Mentioning df separately is good as that is the first ring that we hit when we want to roll out and then rest(msit/prod) can be categorized in same bucket here for the tests.